### PR TITLE
add services and toc

### PR DESCRIPTION
**Pull request by KarishmaGhiya**
Updating for
- monitor-query
- attestation
- core package categorization
- webpubsub categorization
- servicefabric management categorization and readme
- resource subscriptions categorization and readme
- arm-keyvault-readme
- Fixes https://github.com/Azure/azure-sdk-for-js/issues/18308

### CODE REVIEW

**Comment by KarishmaGhiya** [comment]
#fixes https://github.com/Azure/azure-sdk-for-js/issues/18308

**Comment by ramya-rao-a** [comment]
@danieljurek Can you please review?

**Comment by scottaddie** [comment]
Please move this under the "Monitor" service. That's consistent with how we handled Identity:

image

The hierarchy should be "Monitor" > "Client" > "Monitor Query".

**Comment by KarishmaGhiya** [comment]
@danieljurek what is the azure-monitor under monitor service in the TOC? i see it in TOC but not on docs - https://docs.microsoft.com/en-us/javascript/api/overview/azure/monitor?view=azure-node-latest

File: docs-ref-mapping/reference-unified.yml [hunk 1]
```diff
@@ -953,16 +958,16 @@
         - azure-arm-mobileengagement        
     - name: Monitor
       uid: azure.nodejs.sdk.landingpage.services.monitor
-      href: ~/docs-ref-services/{moniker}/monitor.md
+      landingPageType: Service
```
Comment (KarishmaGhiya) [inline docs-ref-mapping/reference-unified.yml:961]:
```suggestion
      href: ~/docs-ref-services/{moniker}/monitor.md
      landingPageType: Service
```

File: docs-ref-mapping/reference-unified.yml [hunk 2]
```diff
@@ -953,16 +958,16 @@
...
+      - name: Monitor Query
+        uid: azure.nodejs.sdk.landingPage.services.monitor.monitorquery
+        href: ~/docs-ref-services/{moniker}/monitor-query-readme.md
         children:
-        - azure-monitor
+        - '@azure/monitor-query'
       - name: Management
         uid: azure.nodejs.sdk.landingPage.services.monitor.Management
-        landingPageType: Service
+        href: ~/docs-ref-services/{moniker}/monitor.md
```
Comment (KarishmaGhiya) [inline docs-ref-mapping/reference-unified.yml:970]:
```suggestion
```

**Comment by KarishmaGhiya** [comment]
@danieljurek Can you review this, I updated with all the information required?

File: docs-ref-mapping/reference.yml [hunk 3]
```diff
@@ -879,10 +884,15 @@
   - name: Monitor
     uid: azure.nodejs.sdk.landingpage.services.monitor
     href: ~/docs-ref-services/latest/monitor.md
+    landingPageType: Service
```
Comment (scbedd) [inline docs-ref-mapping/reference.yml:887]:
```suggestion
```

Comment (scbedd) [reply docs-ref-mapping/reference.yml:887]: You have a `href`, `href` and `landingPageType: Service` are mutually exclusive.

File: docs-ref-mapping/reference-unified.yml (hunk 1 above)
Comment (scbedd) [reply docs-ref-mapping/reference-unified.yml:962]:
```suggestion
```

Comment (scbedd) [reply docs-ref-mapping/reference-unified.yml:962]: You have a href, href and landingPageType: Service are mutually exclusive.

File: docs-ref-mapping/reference-unified.yml [hunk 4]
```diff
@@ -1424,5 +1429,29 @@
     - name: Other
       uid: azure.nodejs.sdk.landingpage.services.other
       landingPageType: Service
-      children:
-      - '**'
+      items:
+      - name: Abort Controller
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:1433]: My only concern, in `reference-unified` we have these `core` packages right. But if I look down below in `reference-latest` or `reference-unified`, I only see maybe two of these?

File: docs-ref-mapping/reference-unified.yml [hunk 5]
```diff
@@ -1424,5 +1429,29 @@
...
+        uid: azure.nodejs.sdk.landingPage.services.other.coreamqp
+        href: ~/docs-ref-services/{moniker}/core-amqp-readme.md
+        children:
+        - '@azure/core-amqp'
+      - name: Core Auth
+        uid: azure.nodejs.sdk.landingPage.services.other.coreauth
+        href: ~/docs-ref-services/{moniker}/core-auth-readme.md
+        children:
+        - '@azure/core-auth'
+       - name: Core Auth
```
Comment (praveenkuttappan) [inline docs-ref-mapping/reference-unified.yml:1448]: Isn't this a duplicate entry of above core auth?

Comment (KarishmaGhiya) [reply docs-ref-mapping/reference-unified.yml:1448]: This looks incomplete. Something is wrong with the commit history

**Comment by KarishmaGhiya** [comment]
@scbedd I updated the PR

File: docs-ref-mapping/reference-unified.yml [hunk 6]
```diff
@@ -917,6 +922,7 @@
       uid: azure.nodejs.sdk.landingpage.services.metricsadvisor
       href: ~/docs-ref-services/{moniker}/ai-metrics-advisor-readme.md
       children:
+      - azure-ai-metrics-advisor
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:925]: I don't think having this present will negatively affect you, but I'm going to let the current publishing build run before merging to be certain.

**Review by scbedd — APPROVED** [review]
Changes look good to me now. Letting the current publishing build finish, then merging to `main`.

File: docs-ref-mapping/reference-unified.yml (hunk 6 above)
Comment (scbedd) [reply docs-ref-mapping/reference-unified.yml:925]: My concern is that these are extraneous, as we already pull this with the entry for `@azure/metrics-advisor`

**Pull request merged**